__tests__/loader.test.js

```diff
@@ -67,6 +67,35 @@ describe('Loader', () => {
     });
   });
 
+  describe('#loadPattern', () => {
+    it('returns a pattern object', () => {
+      const patternPath = '${__dirname}/__fixtures__/patterns/pattern-valid.json';
+      return Loader.loadPattern(patternPath).then(pattern => {
+        expect(pattern).toBeDefined();
+      });
+    });
```

Comment:
Running these tests locally I noticed that this isn't actually returning what it should be because the `patternPath` isn't a template string. This'll fix it and bulk up the code coverage a touch:

```suggestion
    it('returns a pattern object', async () => {
      const patternPath = `${__dirname}/__fixtures__/patterns/pattern-valid.json`;
      await expect(Loader.loadPattern(patternPath)).resolves.toStrictEqual({
        multiLineComment: [
          {
            end: '*/',
            middle: '',
            start: '/*',
          },
        ],
        name: 'Apex',
        nameMatchers: ['.cls'],
        singleLineComment: [
          {
            start: '//',
          },
        ],
      });
    });
```